sympy/utilities/tests/test_lambdify.py

```diff
@@ -581,3 +582,9 @@ def test_issue_2790():
 def test_ITE():
     assert lambdify((x, y, z), ITE(x, y, z))(True, 5, 3) == 5
     assert lambdify((x, y, z), ITE(x, y, z))(False, 5, 3) == 3
+
+def test_10375():
+    import numpy as np
+    m = Min(x, y, z)
+    l = lambdify([x,y,z], m)
```

Comment:
You could use the builtin `min`. 

Anyway, I think when numpy is in the modules we should map `Min` to `lambda x: numpy.amin([x])`.